myStyles.css

```diff
@@ -426,10 +479,44 @@
 
   .menuTextContainer {
     display: none;
+    flex-direction: column;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: 10;
+    background-color: rgb(69, 67, 67);
+    text-align: center;
+  }
+
+  .menuTextContainer a {
+    text-decoration: none;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-size: 40px;
+    text-align: left;
+    padding-left: 10%;
+  }
```

Comment:
- [ ] Ouch! this is close but please make sure to use the proper color, font weight, and spacing between the option menus as there seems to be a bit of discrepancy between your design and the Figma design as shown below. As a front-end developer, you have to pay attention to small details like this as they provided a polished UI.
- Figma design:
  ![image](https://github.com/alexansaa/Personal_Portafolio/assets/104968236/85465a87-b25e-4f90-8524-6749ab7d1c36)
- Your design:
  ![image](https://github.com/alexansaa/Personal_Portafolio/assets/104968236/dff30616-6d8d-42e6-98ad-552f2b8e9b10)
- You can simply achieve this by adding a `font-weight: bold`, a color of `#fff5e1`, and some vertical spacing between the menu options. ✔